source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss

```diff
@@ -165,6 +166,7 @@ $font-family: 'Segoe UI', sans-serif;
             background-color: $hover-background-color;
             color: $hover-color;
             border: $hover-border;
+			cursor: pointer;
```

Comment:
Same as previous comment: I assume we can move `{cursor: pointer}` out of the `hover` state.
nit: spaces